From 2d6a252810c8e334b8718529d72b7ce483049445 Mon Sep 17 00:00:00 2001
From: Ninad Naik <quic_ninanaik@quicinc.com>
Date: Wed, 27 Nov 2024 14:41:24 +0530
Subject: [PATCH] arm64: dts: qcom: sa8775p: Modify correct dt name for SCM

This change is required as this node is looked up and
patched by hypervisor.

Change-Id: I1878059368f36c2fd9b7449dcbdbaa2faab0e9fd
Signed-off-by: Ninad Naik <quic_ninanaik@quicinc.com>
---
 arch/arm64/boot/dts/qcom/sa8775p.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sa8775p.dtsi b/arch/arm64/boot/dts/qcom/sa8775p.dtsi
index 488cdc0d7a5a..f054d3939092 100644
--- a/arch/arm64/boot/dts/qcom/sa8775p.dtsi
+++ b/arch/arm64/boot/dts/qcom/sa8775p.dtsi
@@ -473,7 +473,7 @@ eud_in: endpoint {
 	};
 
 	firmware {
-		scm {
+		qcom_scm {
 			compatible = "qcom,scm-sa8775p", "qcom,scm";
 			qcom,dload-mode = <&tcsr 0x13000>;
 		};
-- 
2.46.0

